src/demo/pages/TooltipPage/examples/Tooltip.Basic.Example.tsx

```diff
@@ -0,0 +1,53 @@
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable */
+
+import {
+  Tooltip,
+  Button
+} from '../../../../index';
+
+export interface ITooltipBasicExampleState {
+  isTooltipVisible?: boolean;
+}
+
+export class TooltipBasicExample extends React.Component<any, ITooltipBasicExampleState> {
+  private _menuButtonElement: HTMLElement;
+
+  public constructor() {
+    super();
+
+    this._onDismiss = this._onDismiss.bind(this);
+
+    this.state = {
+      isTooltipVisible: false,
+    };
+  }
+
+  public render() {
+    let { isTooltipVisible } = this.state;
+
+    return (
+      <div className='ms-TooltipExample'>
```

Comment:
I've got most of this working, though I'm not seeing the Tooltip content being passed through via knockout.

```
<!-- ko component: { 
        name: 'ms-react-tooltip', 
        params: {
            tooltipContent: item().name,
        }
    } -->
<button>This is the KO markup that the tooltip should attach to. But isn't showing up</button>

<!-- /ko -->
```

For this to happen the KO react component needs to pass this KO child into the React child. I'm wondering if that isn't happening, or if KO even supports it.